dev-vcs/git-cola/git-cola-3.5.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+DISTUTILS_SINGLE_IMPL=true
+
+inherit distutils-r1 readme.gentoo-r1 virtualx xdg-utils
+
+DESCRIPTION="The highly caffeinated git GUI"
+HOMEPAGE="https://git-cola.github.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/QtPy[gui,${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/pygments[${PYTHON_USEDEP}]
+	dev-python/send2trash[${PYTHON_USEDEP}]
+	dev-vcs/git"
```

Comment:
`RDEPEND` is inherited in `DEPEND` below, but is it actually all necessary at build time? At least I could just build it fine without having `dev-python/QtPy` and `dev-python/send2trash` installed.

Reply:
I've just pushed some changes, which I hope resolves the problems

actually I already did the BDEPEND change locally but thought I'll wait another review round which was a mistake :( ...